openid-sharedsignals-framework-1_0.md

```diff
@@ -1924,12 +1983,12 @@ subject
   "iss": "https://transmitter.example.com",
   "aud": "receiver.example.com",
   "iat": 1493856000,
+  "sub_id": {
+    "format": "opaque",
+    "id" : "f67e39a0a4d34d56b3aa1bc4cff0069f"
+  },   
   "events": {
     "https://schemas.openid.net/secevent/ssf/event-type/stream-updated": {
-      "subject": {
```

Comment:
I think we said that "subject" in the event will be retained but called out as deprecated.